test/rhsmlib_test/test_pkg_arches.py

```diff
@@ -0,0 +1,80 @@
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+from rhsmlib.facts import pkg_arches
+from mock import patch
+
+
+class TestSupportedArchesCollector(unittest.TestCase):
+
+    @staticmethod
+    def helper_dpkg(*args, **kwargs):
```

Comment:
this almost got me thinking "why is this needed, as `helper_dpkg_foreign()` does the same"... then I realized because this wants to explicitly not support foreign architectures; in this case, I suggest renaming this as `helper_dpkg_no_foreign()`, and adding comments to each function

Reply:
Implemented as suggested.